scripts/zones/Arrapago_Reef/npcs/_jib.lua

```diff
@@ -15,7 +15,7 @@ entity.onTrigger = function(player, npc)
         if player:getXPos() < 8 then
             player:messageSpecial(ID.text.STAGING_GATE_ILRUSI)
             player:messageSpecial(ID.text.STAGING_GATE_INTERACT)
-            player:startEvent(106)
+            player:startCutscene(106)
```

Comment:
Ok, so, when an event returns an option ("wanna open this door?" "wanna use the portal?" etc) the function to use is `:startOptionalCutscene()`

The stagging point does ask you.